### ISSUE TITLE
Support for JSON editor

### ISSUE THREAD

**Issue by kumaresan** [DapperDox/dapperdox#62]
Enabling Editor for feeding individual JSON parameters for 'Body' in POST method, instead of feeding the complete JSON.

https://github.com/jensoleg/swagger-ui

**Comment by kumaresan** [comment]
https://github.com/jensoleg/swagger-ui/blob/master/dist/images/json_editor_integration.png

![json_editor_integration](https://user-images.githubusercontent.com/152916/28161520-69e18d18-67e1-11e7-80d9-4ddf68fa8354.png)

**Comment by zxchris** [comment]
Yes, this is something that we're looking to implement in the explorer. There is much to do, and too few hands!
I've marked it as a feature request.
